### PR TITLE
clarity/typo suggestions

### PR DESCRIPTION
**Pull request by vfatemi**
Please note the point about having a Zoom account associated with the email. It seems necessary and seems worth pointing out if so.

### CODE REVIEW

**Comment by akhmerov** [comment]
The zoom account isn't necessary, what gave an impression that it is?

**Comment by vfatemi** [comment]
One still needs to be able to open and manage the Zoom event. I have had an account so long that I don't know what is possible for people who either don't have the program or don't have an account. 

If the program and/or account are not necessary, then it might be helpful to state that explicitly.

**Comment by akhmerov** [comment]
- I think it should work without an account or the zoom client installed, but it'd be good to test
- Perhaps the detailed speaker instructions belong rather in #179, what do you think? So far I had the mental model that the page should probably have only the birds eye view.

**Comment by vfatemi** [comment]
Regarding birds-eye view -- yes. But, if I were to not have Zoom (as much of an edge-case as that may be), I might read this and think "oh you need Zoom...do I want to deal with that?" Perhaps we can remove reference to Zoom altogether and just say we "schedule the virtual meeting space" or something to that effect?

Also, if the speaker will receive "other instructions" by email then perhaps that would be good to note along with the "gives you host key and registration link". 

Basically, birds-eye view is good, but indicators of more organization and thoroughness under-the-hood will be reassuring to those who read the page closely. (Fixing typos being a first step to that end.)
